mkdocs.template.yml

```diff
@@ -91,9 +91,10 @@ nav:
       - Environment variables: configuration/environment.md
       - Settings: settings.md
   - Integration guides:
-      - Using in Docker: guides/docker.md
-      - Using in GitHub Actions: guides/github.md
-      - Using in pre-commit: guides/pre-commit.md
+      - Using in Docker: guides/integration/docker.md
+      - Using in GitHub Actions: guides/integration/github.md
+      - Using in pre-commit: guides/integration/pre-commit.md
+      - Using with artifactories: guides/integration/artifactories.md
```

Comment:
Personally would drop "Using in" from all of these.

What's "artifactories" here? Isn't Artifactory a specific product from JFrog?

Reply:
Yeah I'm using it as a drop-in term for all such things but maybe there's a better one? We can use "Custom package indexes" but they're not really custom? "Commercial package indexes"?

Reply:
I've been considering dropping the "Using in" too... can't remember where I saw that style.